### ISSUE TITLE
Readme.md needs to be updated to reflect what this fork is capable of 

### ISSUE THREAD

**Issue by Sam-Si** [compsyguy/TekkenBot#3]
Readme.md needs to be updated to reflect what this fork is capable of

**Comment by compsyguy** [comment]
Yeah, I was going to update the readme once I got to a more stable place. Right now I'm trying to figure out the best way to handle everything I want to do.

**Comment by Sam-Si** [comment]
I might be able to help you. If you need

**Comment by compsyguy** [comment]
@Sam-Si Do you have a discord or some other way to chat?

**Comment by Sam-Si** [comment]
@compsyguy Here is my discord sam.si.5268#7640

**Comment by compsyguy** [comment]
Discord friend request sent
